server/server.js

```diff
@@ -50,17 +50,16 @@ app.use((req, res, next) => {
     })
       .then(user => {
         if (!user) {
-          console.log(chalk.cyan('user is not found'))
+          console.log(chalk.cyan('user is not found'));
           //status: user has a cookie id, but login expired
           //next step: user can view pages except Create Route or profile
-          if (whiteList[req.path] === false && user.userType !== 'Admin') {
```

Comment:
had to remove the user.userType condition because if the middleware reaches this point, then user is null

Reply:
Yeah I got this as well, just a few seconds slower than you ;D

Reply:
xD

I'm working on making it so that the page doesn't redirect to login when an admin goes to admin/create -> login -> admin/create